src/main/java/com/codessquad/qna/web/AnswerController.java

```diff
@@ -0,0 +1,67 @@
+package com.codessquad.qna.web;
+
+import com.codessquad.qna.domain.answer.Answer;
+import com.codessquad.qna.domain.question.Question;
+import com.codessquad.qna.domain.user.User;
+import com.codessquad.qna.exception.AnswerNotFoundException;
+import com.codessquad.qna.exception.IllegalUserAccessException;
+import com.codessquad.qna.exception.QuestionNotFoundException;
+import com.codessquad.qna.service.AnswerService;
+import com.codessquad.qna.service.QuestionService;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.*;
+
+import javax.servlet.http.HttpSession;
+
+import static com.codessquad.qna.domain.user.HttpSessionUtils.getUserFromSession;
+
+@Controller
+@RequestMapping("/questions/{questionId}/answers")
+public class AnswerController {
+    private static Logger logger = LoggerFactory.getLogger(AnswerController.class);
+
+    private QuestionService questionService;
+    private AnswerService answerService;
+
+    public AnswerController(QuestionService questionService,
+                            AnswerService answerService) {
+        this.questionService = questionService;
+        this.answerService = answerService;
+    }
+
+    @PostMapping("/")
+    public String createAnswer(@PathVariable Long questionId, String comments,
+                               HttpSession session) {
+        User writer = getUserFromSession(session);
+        Question question = questionService.findBy(questionId);
+        Answer answer = new Answer(writer, question, comments);
+
+        answerService.createAnswer(answer);
+        logger.debug("answer : {}", answer);
+
+        return String.format("redirect:/questions/%d", questionId);
```

Comment:
그냥 덧셈 연산을 하는 쪽이 조금 더 가독성이 좋게 느껴지는게 어떻게 생각하시나요?

```suggestion
        return "redirect:/questions/" + questionId;

Reply:
더하는게 눈에 잘 들어오는거 같습니다😄
말씀해주신 방식으로 변경하겠습니다ㅎㅎ